quantum/config_common.h

```diff
@@ -16,10 +16,4 @@
 
 #pragma once
 
-#ifndef __ASSEMBLER__
-#    include "pin_defs.h"
-#endif
-
-#ifdef AUDIO_ENABLE
```

Comment:
This will still cause problems.  Eg, any of the song defines will not use the defined values.  

Short of moving the song list to the `CONFIG_H` variable in makefiles... Any of these will be broken/not respected: 
https://docs.qmk.fm/#/feature_audio?id=audio-config

Reply:
Then it would have already be broken by #19714 and not this PR.

Reply:
Then it would have been further broken by this PR, yes. 

So now, every keymap had custom configuration for those sounds is now broken. Awesome.

For reference, the sounds will still play, but will ONLY be the default sounds, and CANNOT be overwritten.